deps.ts

```diff
@@ -1,11 +1,11 @@
-export { relative, resolve } from "https://deno.land/std@v0.55.0/path/mod.ts";
-export { readJson } from "https://deno.land/std@v0.55.0/fs/read_json.ts";
+export { relative, resolve } from "https://deno.land/std@v0.61.0/path/mod.ts";
+export { readJson } from "https://deno.land/std@v0.61.0/fs/read_json.ts";
```

Comment:
Same as above, lets bump std to 0.63.0
```suggestion
export { relative, resolve } from "https://deno.land/std@v0.63.0/path/mod.ts";
export { readJson } from "https://deno.land/std@v0.63.0/fs/read_json.ts";
```